jetty9/src/main/docker/Dockerfile

```diff
@@ -39,8 +39,9 @@ RUN mkdir webapps $TMPDIR \
  && chown -R jetty:jetty $JETTY_BASE $TMPDIR
 
 # Start Jetty directly
-COPY app-env.bash /
-RUN chmod +rx /app-env.bash
+COPY setup-env-ext.bash /tmp/
+RUN cat /tmp/setup-env-ext.bash >> /setup-env.bash \
+ && rm /tmp/setup-env-ext.bash
```

Comment:
I don't think we really need to remove the file. It's already going take up a tiny amount of space in the image because the copy is a separate layer.